### DISCUSSION TITLE
Support Cloud Object Storage ?

### DISCUSSION THREAD

**Discussion by yanming-zhang** [juicedata/juicefs#2616] [General]
Does juicefs support JD Cloud Object Storage?

**Comment by SandyXSD** [comment]
You can probably use it with `S3` protocol. see: https://docs.jdcloud.com/cn/object-storage-service/introduction-3

**Comment by yanming-zhang** [comment]
> https://docs.jdcloud.com/cn/object-storage-service/introduction-3

thanks
